pages/index.js

```diff
@@ -19,13 +23,18 @@ export default function HomePage({ dates, places }) {
           src={"/pictures/Header-Home.png"}
           style={{ objectFit: "contain" }}
         />
-        <ShowDatesList dates={dates} places={places} />
-        <StyledNavigation color="darkgreen" navigationlink="/lists/plantlist">
-          Alle Pflanzen anzeigen
-        </StyledNavigation>
-        <StyledNavigation color="darkred" navigationlink="/lists/placelist">
-          Alle Standorte anzeigen
-        </StyledNavigation>
+
+        <ShowDatesList />
+        <StyledCardLink href={"lists/placelist"}>
+          <StyledCard backgroundcolor="globalPlaceBackgroundColor">
+            Alle Standorte anzeigen
+          </StyledCard>
+        </StyledCardLink>
+        <StyledCardLink href={"lists/plantlist"}>
+          <StyledCard backgroundcolor="globalPlantBackgroundColor">
+            Alle Pflanzen zeigen
+          </StyledCard>
+        </StyledCardLink>
```

Comment:
Why not style the link directly or do you need the StyledCard really? 🤔

Reply:
I started a trial with Flex and Grid without getting a similar result in the short term. Therefore I leave it for the time being and take care of it with refactoring.